docs/reference/glossary-of-terms.md

```diff
@@ -0,0 +1,227 @@
+
+#### Individual
+An [Ontology elements](#Ontology-element) that represents an instance of a class. For example, the instance "John" or "John's heart". Note that instances are not commonly represented in ontologies.
+
+#### Information Content
+A measure of how informative an ontology concept is; broader concepts are less informative as they encompass many things, whereas more specific concepts are more unique. This is usually measured as ``-log2(Pr(term))``. The method of calculating the probability varies, depending on which predicates are taken into account (for many ontologies, it makes sense to use part-of as well as is-a), and whether the probability is the probability of observing a descendant term, or of an entity annotated using that term.
+
+#### Interface
+A programmatic abstraction that allows us to focus on *what* something should do rather than *how* it is done.
+
+#### Jaccard Similarity
+A measures of the similarity between two sets of data to see which members are shared and distinct.
+
+#### KGCL
+Knowledge Graph Change Language (KGCL) is a [data model](#Data model) for communicating desired changes to an ontology. It can also be used to communicate differences between two ontologies. See [KGCL docs](https://github.com/INCATools/kgcl).
+
+#### Knowledge Graph
+A network of real-world entities (i.e., objects, events, situations, and concepts) that illustrates the relationships between them. Knowledge graphs (in relation to ontologies) are thought of as real data built using an ontology as a framework.
+
+#### Label
+Usually refers to a human-readable label corresponding to the `rdfs:label` [predicate](#predicate). Labels are typically unique per ontology. In OBO Format and in the bio-ontology literature, labels are sometimes called [Names](#Name). Sometimes in the machine learning literature, and in databases such as Neo4J, "label" actually refers to a [Category](#Category).
+
+#### Lutra
+[Lutra](https://ottr.xyz/#Lutra) is the open source reference implementation of the OTTR templating language.
+
+#### Mapping
+See [SSSOM]('SSSOM')
+
+#### Materialised
+The process of calculating the implicit statements in an ontology - making inferred axioms explicit by asserting them.
+
+#### Name
+Usually synonymous with [Label](#Label), but in the formal logic and OWL community, "Name" sometimes denotes an [Identifier](#Identifier)
+
+#### Named Individual
+An [Individual](#Individual) that is given an explicit name that can be used in any ontology to refer to the same object.
+
+#### Nodes
+[Terms](#Term) represented in a [graph](#Graph)
+
+#### Object
+The "right" side of a [Triple](#Triple).
+
+#### ObjectProperty
+A simple association between two terms, also see [predicate](#predicate) & [Relationship](#Relationship). 
+
```

Comment:
I think this needs work.  I'm really not sure about the loose use of term throughout this glossary.  I think easier to use OWL terminology of classes and individuals etc & have an entry for term explaining its general usage.